2-mandatory/1-create-functions.js

```diff
@@ -24,17 +21,22 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
-}
+const modifyStr = (names) => names.trim().replace("/", "").toLowerCase();
+
+const tidyUpString = (arrOfString) => arrOfString.map(modifyStr);
 
 /*
 Write a function that:
 - Takes an array and an index as input.
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
-}
+const remove = (arr, index) => arr.filter((element) => element !== arr[index]);
```

Comment:
Perfect answer! I also really like how you wrote `index` and  `element` as opposed to `i` and `e` like many other people doo. When these functions become 10-20 lines long, readability is going to be super important, and writing code like this is so much better! 🎉